src/target/stm32f1.c

```diff
@@ -94,6 +94,7 @@ static bool stm32f1_mass_erase(target_s *target);
 
 #define DBGMCU_IDCODE    0xe0042000U
 #define DBGMCU_IDCODE_F0 0x40015800U
```

Comment:
These need re-formatting so the `0x` of the constants align with the start of the `DBGMCU_IDCODE_GD32E5` one (use spaces, or run clang-format on this one file)

Reply:
If you approve of the macro name, then I'll align (touch adjacent lines) with spaces like clang-format-15 suggests.

Reply:
We do, the macro name seems good